config/hsc/processCcd.py

```diff
@@ -13,6 +13,12 @@
 config.charImage.measurePsf.starSelector["objectSize"].fluxMin = 4000
 config.calibrate.refObjLoader.load(os.path.join(hscConfigDir, "filterMap.py"))
 
+# Set to same defaults used in HSC
```

Comment:
This *is* HSC!

(Same as on meas_mosaic: in decades to come, that comment will be more confusing than helpful. Maybe better if you can refer to a specific HSC data release this were used for?)